docs/confounding-variables.md

```diff
@@ -0,0 +1,40 @@
+# Confounding Variables
+
+A confounding variable is a factor that influences independent variable and dependant variable. 
+A confounding variable can mix with the effect of an independent variable on the dependent variable thus leading to erronous conclusions about the experiment.
+
+At any time during your experiment, the data manager allows you to capture confounding variables. 
+
+### Define confounding variables in your experiment
+
+In order to work with confounding variables, first you must define them in your experiment. 
+Confounding variables are experiment specific. Confounding variables do not interact with
+information outside your experiment.
+
+To define a new confounding variable, navigate to your experiment, open the _Confounding Variables_ tab, click on **Add confounding variables**. In case confounding variables are defined already, click **Add** in the top right corner of the tab.
```

Comment:
Since we dont know if it will remain a tab, consider leaving out UI details in the description.

Reply:
I can remove the fact that it is a tab. Although I am unsure how much guidence the user needs here. I could simply write "Navigate to your experiment and change the confounding variables there". But I fear this level of detail is needed here.